.env

```diff
@@ -0,0 +1,11 @@
+# Flask server configuration.
+FLASK_APP=todo_app/app
+FLASK_ENV=development
+
+# Change the following values for local development.
+TRELLO_API_KEY = "3c0b3b2ac6629a89756c048878152c34"
+
+TRELLO_API_SECRET = "ca21e70f253ffbd83fc016a2bb18fc94c874e5e74aeeef4f8511914c9574a839"
+TRELLO_TOKEN = "94997e12c3e2eb3a5145759901446130f3e85e6dab4909664cc11ee08ed83114"
+TRELLO_BOARD_ID = "gCbe0F6f"
+TRELLO_USERNAME = "christophersweeney8"

```

Comment:
Separate to the security issue, there's a slight syntax issue. Flask will cope with what you've done, but technically you shouldn't have these spaces and quotation marks. In a later module, a tool called Docker will struggle with your formatting.

E.g. this line should be `TRELLO_USERNAME=christophersweeney8`.

This is not a Python file so don't use Python syntax.